packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.tsx

```diff
@@ -139,7 +154,7 @@ export class HoverCard extends BaseComponent<IHoverCardProps, IHoverCardState> {
 
     this._dismissTimerId = this._async.setTimeout(() => {
       if (!(this.props.sticky && type === 'mouseleave')) {
-        const rect = this._hoverCard.getBoundingClientRect();
+        const rect = this._expandingCard.element.getBoundingClientRect();
```

Comment:
This will be only called when the card is open and trying to dismiss, and not when a user is casually moving the cursor over targets. 

Callbacks in the event of `mouseLeave` and `mouseEnter` manage the state of `isHoverCardVisible`. But this technique fails when the card expands and is overlapping the target; wherein upon cursor move `mouseEnter` from card is called before `mouseLeave` from the target, resulting in end state of `isHoverCardVisible` to be false. 

If there is a cheaper way to know if the card is going to overlap the target, then we can short-circuit the measure.